src/util_libc.rs

```diff
@@ -141,11 +141,7 @@ impl Weak {
 }
 
 cfg_if! {
-    if #[cfg(any(target_os = "linux", target_os = "emscripten"))] {
-        use libc::open64 as open;
-    } else {
-        use libc::open;
-    }
+    use libc::open;
```

Comment:
You can simply remove `cfg_if` and use `libc::open` at call sites (I think we have only one such place).